docs/documentation/upgrading/topics/keycloak/changes-22_0_0.adoc

```diff
@@ -74,3 +77,56 @@ NOTE:: When the auto-build runs, the build time options will be in effect for al
 In previous releases the `export` and `import` commands allowed runtime parameters like, for example, a database URL only in configuration files or environment variables.
 Starting with this release, those runtime parameters are now available on the command line as well.
 Use the `--help` option to find out about the supported parameters.
+
+= Renamed Keycloak Admin client artifacts
+
+After the upgrade to Jakarta EE, artifacts for Keycloak Admin clients were renamed to describe the reality more accurately, with consideration for long-term maintainability.
+We still provide two separate Keycloak Admin clients, one with Jakarta EE and the other with Java EE support.
+
+We stopped releasing the `org.keycloak:keycloak-admin-client-jakarta` artifact.
+The default one for Keycloak Admin client with Jakarta EE support is `org.keycloak:keycloak-admin-client` (since version 22.0.0).
```

Comment:
```suggestion
The default one for the Keycloak Admin Console with Jakarta EE support is `org.keycloak:keycloak-admin-client` (since version 22.0.0).
```